api/swagger.yaml

```diff
@@ -2096,6 +2096,12 @@ definitions:
       ForceUpdate:
         description: "A counter that triggers an update even if no relevant parameters have been changed."
         type: "integer"
+      Runtime:
+        description: "Runtime is the type of runtime specified for the task executor."
+        type: "string"
+      RuntimeData:
+        description: "RuntimeData is the payload sent to be used with the runtime for the executor."
```

Comment:
I don't think it should be exposed - it's internal data, however we decide to represent that.

The only user is going to be internal code, so I don't think we need to expose that at all